### PR TITLE
Butadiene_Dimerization added to thermo library, kinetic library, and training reaction

### PR DESCRIPTION
**Pull request by hwpang**
Butadiene dimerization reactions based on CBS-QB3 with rotors from Duminda Ranasinghe and Hao-Wei Pang have been added to the kinetic library and the thermo library at Butadiene_Dimerization.

The 1,3-butadiene <=> cyclobutene reaction has been added to the training reactions in the Intra_2+2_cycloaddition_Cd kinetic family.

The C=CC=C + C=CC=C <=> CH2]C=CCCC=C[CH2] reaction has been added to the training reactions in the 1,4_Linear_birad_scission kinetic family.

The C=CC=C + C=CC=C <=> C=CC1CC=CCC1 reaction has been added to the training reactions in the Diels-Alder reaction kinetic family.

The C=CC=C + C=CC=C <=> C=CC1CCC1C=C reaction has been added to the training reactions in the 2+22+2_cycloaddition_Cd kinetic family.

The C=CC1CCC1C=C <=> C1=CCCC=CCC1 reaction has been added to the training reaction in the 6_membered_central_C-C_shift kinetic family.

The [CH2]C=CCCC=C[CH2] <=> C=CC1CC=CCC1 reaction has been added to the training reaction in the Birad_recombination kinetic family.

The [CH2]C=CCCC=C[CH2] <=> C1=CCCC=CCC1 reaction has been added to the training reaction in the Birad_recombination kinetic family.

The C=C[CH]CC[CH]C=C <=> C=CC1CCC1C=C reaction has been added to the training reaction in the Birad_recombination kinetic family.

### CODE REVIEW

File: input/kinetics/families/Birad_recombination/training/reactions.py [hunk 1]
```diff
@@ -136,3 +136,54 @@
...
+    kinetics = Arrhenius(
+        A = (3.49424e+11, 's^-1'),
+        n = -0.0806221,
+        Ea = (12.5388, 'kJ/mol'),
+        T0 = (1, 'K')),
+    rank = 5,
+    shortDesc = u"Calculation performed by Duminda Ranasinghe and Hao-Wei Pang, March 2019, CBS-QB3 level of theory",
+    longDesc = 
+    u"""
+    Location of calculations Pharos/home/hwpang/Dow/Pure/QM/Reactions/Dimer
```
Comment (mjohnson541) [inline input/kinetics/families/Birad_recombination/training/reactions.py]: Remove the server location information from all of these files, it's easy for a Green group member to get separately and useless for someone outside the group.  It also makes it look like we built the database to be only used by the Green group, which is undesirable.  The names of the people that calculated the values and the QM methods and rotor treatments used should be sufficient.

**Comment by mliu49** [comment]
What's the status on this PR?

**Comment by hwpang** [comment]
I've refitted three reactions using the same CBS-QB3 results with atom additivity correction set to Ture in Arkane. These three reactions were accidentally set to False in the previous fitting. The kinetic library and the training reactions in corresponding kinetic families were all updated.

**Comment by mliu49** [comment]
@hwpang, if you would like this to be included in the release, please rebase and ask Matt for final review.

**Review by kspieks — APPROVED** [review]
Thanks for adding these reactions! LGTM. I'm happy to merge as soon as the tests pass.

**Comment by kspieks** [comment]
Thanks for fixing the index issues. Hopefully the tests pass now

**Pull request merged**